### DISCUSSION TITLE
net_flow_tcp_begin missing in logs

### DISCUSSION THREAD

**Discussion by Ziphone** [aquasecurity/tracee#3921] [Questions and Help]
## Environment
Azure AKS, K8s: 1.27, OS: Ubuntu 22.04.4 LTS, Kernel: 5.15.0-1057-azure
Installation: Helm, chart: tracee, version 0.20.0

## Issue
My cluster hosts numerous services, which is reflected by a lot of `net_tcp_connect`, `security_socket_connect`, and `net_packet_http_request` events in the Tracee logs. Despite including `net_flow_tcp_begin` in my Policy resource, I don't get any such events. I have also tested `tracee --output json --events net_flow_tcp_begin`, with no luck.

Is this new event enabled? Could there be any gotchas I have missed out on?

**Comment by littlestar1998** [comment]
Have you solved it yet? I'm having the same problem.
